scripts/partition_manager/partition_manager.rst

```diff
@@ -38,6 +42,25 @@ Container partitions
 
 The start addresses and sizes of image partitions are used in the preprocessing of the linker script for each image.
 
+RAM partition types
+=====================
+
+Image RAM partitions
+   An image RAM partition consists of all RAM that is not defined as a permanent image RAM partition or placeholder RAM partition.
+   It is the default RAM partition associated with an image.
```

Comment:
We could mention as an aside that this is the memory area available when linking the image. This could be mentioned for flash image partitions as well. It's info for advanced users, but will be very useful for those users.

Reply:
Added this info here, and on the section about permanent image RAM partitions as well.